public/css/skeleton.css

```diff
@@ -610,16 +608,26 @@ there.
 */
 
 /* Larger than mobile */
-@media (min-width: 400px) {}
+@media (min-width: 400px) {
+  /* To fill */
+}
 
 /* Larger than phablet (also point when grid becomes active) */
```

Comment:
phablet? Ok definitely learned a new word today :-)

Reply:
I actually didn't realise it was a commonly used word until now and seeing that the RWD tester extension I use also used that term :grin:.